lib/msal-browser/src/app/PublicClientApplication.ts

```diff
@@ -661,11 +689,23 @@ export class PublicClientApplication {
         return authCodeRequest;
     }
 
-    private generateLogoutRequest(logoutRequest?: EndSessionRequest): EndSessionRequest {
-        return {
-            ...logoutRequest,
-            postLogoutRedirectUri: this.getPostLogoutRedirectUri(logoutRequest ? logoutRequest.postLogoutRedirectUri : "")
+    /**
+     * Initializer for the logout request.
+     * @param logoutRequest 
+     */
+    private initializeLogoutRequest(logoutRequest?: EndSessionRequest): EndSessionRequest {
+        const validLogoutRequest = {
```

Comment:
what if logout request is null, will `...logoutRequest` throw?

Reply:
@jasonnutter  cc

Reply:
I think it will throw

Reply:
yeah, just tested it, spread operator can't be called on null

Reply:
Spreading `null` should work inside objects (but not arrays).